test/test_linalg.py

```diff
@@ -5211,6 +5211,39 @@ def test_householder_product_errors_and_warnings(self, device):
             with self.assertRaisesRegex(RuntimeError, "Expected all tensors to be on the same device"):
                 torch.linalg.householder_product(reflectors, tau)
 
+    def _test_fn_out_strided(self, fn, from_4rank_tensors_fn, A):
+        out = fn(A)
+
+        def make_4rank_like(t):
+            fill_with = [3] * (4 - len(t.shape))
+            shape = fill_with + list(t.shape)
+            # Create a 4-rank tensor with non-C-nor-F-contiguous strides.
+            return torch.empty(shape, dtype=t.dtype).contiguous(memory_format=torch.channels_last)
+
+        out_ = from_4rank_tensors_fn([make_4rank_like(o) for o in out])
+        strides_ = list(o.stride() for o in out_)
+
+        fn(A, out=out_)
+
+        for i in range(len(out)):
+            # Check whether the results are equal for the 'out' overload.
+            self.assertEqual(out[i], out_[i])
+            # Check whether the strides were preserved after the call.
+            self.assertEqual(strides_[i], out_[i].stride())
+            # Check whether the original strides are different from the generated ones.
+            self.assertNotEqual(out[i].stride(), out_[i].stride())
```

Comment:
I believe these things are already tested in the `OpInfo` tests in
https://github.com/pytorch/pytorch/blob/c83f8ee46ab057a8bce7ed7ba7ceccba5c388d8f/test/test_ops.py#L676-L684

Reply:
Ah, you are right! Thanks for pointing that out.